test/plots/wheelPlotTests.ts

```diff
@@ -0,0 +1,304 @@
+///<reference path="../testReference.ts" />
+
+describe("Plots", () => {
+  describe("WheelPlot", () => {
+    describe("basic usage", () => {
+      let SVG_WIDTH = 400;
+      let SVG_HEIGHT = 500;
+      let svg: d3.Selection<void>;
+      let wheelPlot: Plottable.Plots.Wheel<number, number>;
+      let rScale: Plottable.Scales.Linear;
+      let tScale: Plottable.Scales.Linear;
+
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+        rScale = new Plottable.Scales.Linear();
+        tScale = new Plottable.Scales.Linear();
+        tScale.domain([0, 360]);
+        wheelPlot = new Plottable.Plots.Wheel();
+        wheelPlot.innerRadius((d) => d.r1, rScale);
+        wheelPlot.outerRadius((d) => d.r2, rScale);
+        wheelPlot.startAngle((d) => d.t1, tScale);
+        wheelPlot.endAngle((d) => d.t2, tScale);
+      });
+
+      it("renders correctly with no data", () => {
+        assert.doesNotThrow(() => wheelPlot.renderTo(svg), Error);
+        assert.strictEqual(wheelPlot.width(), SVG_WIDTH, "plot has been allocated width");
+        assert.strictEqual(wheelPlot.height(), SVG_HEIGHT, "plot has been allocated height");
+
+        svg.remove();
+      });
+
+      it("the accessors properly access data, index and Dataset", () => {
+        let data = [
+          {r1: 0, r2: 1, t1: 0, t2: 180 },
+          {r1: 1, r2: 2, t1: 180, t2: 360 }];
+        let dataset = new Plottable.Dataset(data);
+        wheelPlot.addDataset(dataset);
+        wheelPlot.renderTo(svg);
+
+        let slices = wheelPlot.selections();
+        let path1 = d3.select(slices[0][0]).attr("d");
+        let path2 = d3.select(slices[0][1]).attr("d");
+
+        let arc1 = d3.svg.arc().innerRadius(rScale.scale(0)).outerRadius(rScale.scale(1))
+                               .startAngle(tScale.scale(0)).endAngle(tScale.scale(180));
+        let arc1Path = (<any> arc1)();
```

Comment:
Is the cast needed because the signature expects a `d`? You can pass in a dummy datum in that case, or `null`.